docs/static/monitoring/monitoring-output.asciidoc

```diff
@@ -20,28 +20,27 @@ Otherwise, monitoring data might be routed in different ways or to different pla
 [[logstash-monitoring-default]]
 ===== Default Configuration
 
-If a Logstash node does not explicitly define an {monitoring} output setting, 
+If a Logstash node does not explicitly define a monitoring output setting, 
 the following default configuration is used:
 
 [source,yaml]
 ---------------------------------------------------
-xpack.monitoring.elasticsearch.hosts: [ "http://localhost:9200" ]
+monitoring.elasticsearch.hosts: [ "http://localhost:9200" ]
 ---------------------------------------------------
 
-All data produced by {monitoring} for Logstash is indexed in the monitoring 
+All data produced by Logstash monitoring is indexed in the monitoring 
```

Comment:
Need to verify whether this is still correct

Reply:
TBD